Software/src/ApiServer.cpp

```diff
@@ -501,31 +499,31 @@ void ApiServer::clientProcessCommands()
 			for (int i = 0; i < curentColors.count(); i++)
 			{
 				QColor color(curentColors[i]);
-				result += QString("%1-%2,%3,%4;").arg(i).arg(color.red()).arg(color.green()).arg(color.blue());
+				result += QStringLiteral("%1-%2,%3,%4;").arg(i).arg(color.red()).arg(color.green()).arg(color.blue());
 			}
-			result += "\r\n";
+			result += QStringLiteral("\r\n");
 		}
 		else if (cmdBuffer == CmdGetFPS)
 		{
 			API_DEBUG_OUT << CmdGetFPS;
 
-			result = QString("%1%2\r\n").arg(CmdResultFPS).arg(lightpack->GetFPS());
+			result = QStringLiteral("%1%2\r\n").arg(CmdResultFPS).arg(lightpack->GetFPS());
 		}
 		else if (cmdBuffer == CmdGetScreenSize)
 		{
 			API_DEBUG_OUT << CmdGetScreenSize;
 
 			QRect screen = lightpack->GetScreenSize();
 
-			result = QString("%1%2,%3,%4,%5\r\n").arg(CmdResultScreenSize).arg(screen.x()).arg(screen.y()).arg(screen.width()).arg(screen.height());
+			result = QStringLiteral("%1%2,%3,%4,%5\r\n").arg(CmdResultScreenSize).arg(screen.x()).arg(screen.y()).arg(screen.width()).arg(screen.height());
 		}
 		else if (cmdBuffer == CmdGetCountMonitor)
 		{
 			API_DEBUG_OUT << CmdGetCountMonitor;
 
 			int count = QGuiApplication::screens().count();
 
-			result = QString("%1%2\r\n").arg(CmdResultCountMonitor).arg(count);
+			result = QStringLiteral("%1%2\r\n").arg(CmdResultCountMonitor).arg(count);
```

Comment:
When did you swtich to .arg(x,y) and when stick to .arg(X).arg(Y) ?

Reply:
`(QString1, QString2...)` (up to 9 i think)
`arg(type1).arg(type2)...` which is probably equal to `(QString(type1), QString(type2)...)` allocation wise